composer.json

```diff
@@ -43,9 +44,11 @@
         }
     },
     "scripts": {
-        "test": "vendor/bin/phpunit"
+        "test": [
+            "vendor/bin/phpstan analyse --ansi",
+            "vendor/bin/psalm --taint-analysis",
+            "vendor/bin/phpunit"
+        ],
+        "tests": "@test"
```

Comment:
What is the advantage of the `tests` script in comparison with `test`?

Reply:
The main idea here is to be semantically accurate as multiple tests are performed, not just a single one.

On the other hand, I wanted to keep previously introduced script, since I'm not totally sure whether this change should be treat as a BC one. I would rather consider it as not the BC one, since the library's public API hasn't been changed, therefore I'm in favor to drop `test` script in favor of `tests` one.

Any objections?

Reply:
> drop test script in favor of tests one.

Let's do this

Reply:
done